_posts/en/newsletters/2022-10-26-newsletter.md

```diff
@@ -0,0 +1,270 @@
+          refactored code to accomplish a desirable goal.
+
+        - Produce high-level explainers for long-running projects
+          describing the state before the project, the current progress,
+          what it will take to accomplish the outcome, and the benefits
+          that will provide to users
+
+        - Form working groups with those who are interested in the same
+          projects or code subsystems
+
+- **Ephemeral anchors:** Greg Sanders followed up previous discussion
+  about v3 transaction relay (see [Newsletter #220][news220 ephemeral])
+  with a [post][sanders ephemeral] to the Bitcoin-Dev mailing containing
+  a proposal for a new type of [anchor output][topic anchor outputs].  A
+  v3 transaction could pay zero fees but contain an output paying the
+  script `OP_TRUE`, allowing anyone to spend it under the consensus
+  rules in a child transaction.  The zero-fee parent transaction would
+  only be accepted if it was part of a transaction package which also
+  contained the child transaction spending the OP_TRUE output.
+
+    Described advantages of this proposal is that it eliminates the need
+    to use one-block relative timelocks (called `1 OP_CSV` after the
+    code used to enable them) to prevent [transaction pinning][topic
+    transaction pinning] and allows anyone to fee bump the parent
+    transaction (similar to an earlier [fee sponsorship][topic fee
+    sponsorship] proposal).
+
+    Jeremy Rubin [replied][rubin ephemeral] in support of the proposal
+    but noted that it doesn't work for any contract that can't use v3
+    transactions.  Several other developers also discussed the concept,
+    all of them seeming to find it appealing as of this writing.
+
+## Selected Q&A from Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 24.0 RC2][] is a release candidate for the
+  next version of the network's most widely used full node
+  implementation.  A [guide to testing][bcc testing] is available.
+
+  **Warning:** this release candidate includes the `mempoolfullrbf`
+  configuration option which several protocol and application developers
+  believe could lead to problems for merchant services as described
+  in the newsletters for this week and last week.
```

Comment:
Thanks for adding a note about mempoolfullrbf! I hope it causes people to test rc2 and consider their zeroconf setups. I also hope it doesn't cause people to think Bitcoin Core is deliberately/thoughtlessly putting anyone at risk.